common/data/sentencePlanningApi.js

```diff
@@ -44,6 +44,11 @@ const updateSentencePlanObjective = async (sentencePlanId, objectiveId, objectiv
   return putData(path, token, objective)
 }
 
+const getSentencePlanNeeds = async (sentencePlanId, token) => {
```

Comment:
Interesting general thought raised by John; `async` in these functions serves no real purpose. The only level which needs the async and await is the end consumer, the rest just return the promise.

Reply:
Ah, very true - I'm guessing they are all in there because of a bit of historical refactoring oversight. I've taken them out - everything still working fine.